templates/OperationSnippet.stencil

```diff
@@ -5,10 +5,10 @@
 ///    - completionHandler: A completion handler that receives a status code on
 ///     success.
 public func {{ op.name }}(
-    {% for param in op.params %}
+    {% for param in op.signatureParams %}
     {{ param.name }}: {{ param.type }}{{ param.defaultValue }},
     {% endfor %}
-    withOptions options: {{ model.name }}Options? = nil,
+    withOptions {{ op.clientMethodOptionsViewModel.variableName }}: {{ op.clientMethodOptionsViewModel.name }}? = nil,
```

Comment:
Especially since this is an internal label, there's really no reason NOT to use a static name. It simplifies the stencil files greatly.

Reply:
Changed per feedback.